index.html

```diff
@@ -392,7 +394,7 @@
   ctx.font = radius*0.20 + "px arial";
   ctx.textBaseline="middle";
   ctx.textAlign="center";
-  for(num = (max % incr === 0) ? max - incr : max - max % incr; num >= 0; num -= incr){

```

Comment:
It looks like this change is reverting another change to this line that has already been merged into master. I think it can get fixed by rebasing the change against master to pull all new master changes into this feature branch, but I don't know how that's done with the GitHub UI... It may get resolved automatically when you go to merge the pull request, but I'm not familiar enough with how this works. Does anyone else have a better idea of whether this is an issue and how to deal with it?